src/common/info/info.ts

```diff
@@ -77,9 +77,9 @@ export function resolvePropsFromComponentSel(
   return propOpts;
 }
 
-const handleValueNotFound = <T = any>(value: T) => {
+const handleValueNotFound = <T = any>(value: T, showNotFound = true) => {
   const current = {
-    label: `${value} (not found)`,
+    label: `${value} ${showNotFound ? '(not found)' : ''}`,
```

Comment:
It doesn’t make sense to me to show "(not found)" if we [explicitly](https://github.com/grafana/grafana-iot-twinmaker-app/blob/6083bfa221863bde301500f74bd6a9f7a32b140d/src/datasource/components/ConfigEditor.tsx#L76) allow custom values in the “workspaces” input? Not sure about the reason for this, as I would expect the “save and test” to fail in case the provided workspaces isn’t in the list, but there could be a use case for this.